From 8bc7bdac12996a5576c5e3ff11bb48db5d08bcdf Mon Sep 17 00:00:00 2001
From: Felipe Balbi <felipe.balbi@linux.intel.com>
Date: Fri, 13 May 2016 10:11:59 +0300
Subject: [PATCH 03/41] usb: dwc3: gadget: remove udelay(1) when sending ep
 cmds

When we send an endpoint command, we want that to
complete as soon as possible, so let's remove the
unnecessary udelay(1) call.

Signed-off-by: Felipe Balbi <felipe.balbi@linux.intel.com>
(cherry picked from commit b43bba96b9036e42b2c2c71ee15e1f77b0b37aec)
Signed-off-by: Janusz Dziedzic <januszx.dziedzic@intel.com>

Change-Id: I9c389b7399d90ae8c7e8120b21d7df273e2265fe
Tracked-On: https://jira01.devtools.intel.com/browse/OAM-34814
Reviewed-on: https://android.intel.com:443/519773
---
 drivers/usb/dwc3/gadget.c | 2 --
 1 file changed, 2 deletions(-)

diff --git a/drivers/usb/dwc3/gadget.c b/drivers/usb/dwc3/gadget.c
index f6fe8c9..34174a5 100644
--- a/drivers/usb/dwc3/gadget.c
+++ b/drivers/usb/dwc3/gadget.c
@@ -391,8 +391,6 @@ int dwc3_send_gadget_ep_cmd(struct dwc3 *dwc, unsigned ep,
 			ret = -ETIMEDOUT;
 			break;
 		}
-
-		udelay(1);
 	} while (1);
 
 	if (unlikely(susphy)) {
-- 
1.9.1

